addons/spectate/functions/fnc_initSpectatorUnit.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: AACO
+ * Function used to start the spectate script from the placeable spectate unit
+ * Should only be called from the spectate unit init event dispacher
+ *
+ * Arguments:
+ * 0: Spectate unit <OBJECT>
+ *
+ * Examples:
+ * [player] call potato_spectate_fnc_initSpectatorUnit;
+ *
+ * Public: No
+ */
+
```

Comment:
`if (!hasInterface) exitWith {};`
might need this or dedi server might wait forever, not sure

Reply:
Init should be local to the unit, but it's a good saftey check

Reply:
Init is not local, so this is a very good check